tools/restart-services

```diff
@@ -13,10 +16,11 @@ then
 fi
 
 sleep 3
-sudo /etc/init.d/elasticsearch start
-sudo /etc/init.d/gearman-job-server start
-sudo rm /tmp/archivematicaMCP*
+sudo service elasticsearch start
+sudo service gearman-job-server start
 sudo start fits
+sudo service nginx start
+sudo service uwsgi start
 sudo start archivematica-mcp-server
 sudo start archivematica-mcp-client
-sudo apache2ctl restart
+sudo service apache2 start
```

Comment:
I believe there are cases where it is necessary to stop apache, at least. For example, if you want to update the MCP database, and apache is still running, it is possible that there are open database connections, and any sql like 'drop database' will hang.